build.assets/robotest/nightly_config.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source $(dirname $0)/utils.sh
+
+# UPGRADE_MAP maps gravity version -> list of linux distros to upgrade from
+declare -A UPGRADE_MAP
+
+UPGRADE_MAP[$(latest_release $(branch 7.0.x))]="centos:7 debian:9 ubuntu:18" # compatible LTS version
+UPGRADE_MAP[7.0.0]="ubuntu:16"
+UPGRADE_MAP[$(latest_release $(branch 6.3.x))]="redhat:7" # compatible non-LTS version
+# UPGRADE_MAP[6.3.0]="ubuntu:16"  # disabled due to https://github.com/gravitational/gravity/issues/1009
+UPGRADE_MAP[$(latest_release $(branch 6.2.x))]="redhat:7" # compatible non-LTS version
```

Comment:
These should all use `recommended_upgrade_tag` instead of `latest_release` to make sure this still does the right thing when it is several months old. and 7.0.x may have etcds that are more current than this commit supports.

Reply:
Fixed in ab36936.